sw/device/tock/chips/earlgrey/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+//! Drivers and chip support for the Ibex soft core.
+
+#![feature(asm, concat_idents, const_fn, naked_functions)]
+#![feature(in_band_lifetimes)]
+#![no_std]
+#![crate_name = "earlgrey"]
+#![crate_type = "rlib"]
```

Comment:
Is there a reason you need these? Normally Cargo provides these to the compiler.